packages/web/src/utils/route.ts

```diff
@@ -34,6 +34,10 @@ export const fullAiPage = (handle: string) => {
   return `${fullProfilePage(handle)}/ai`
 }
 
+export const fullCommentHistoryPage = (handle: string) => {
+  return `${fullProfilePage(handle)}/comments`
```

Comment:
I think this should use `profilePage` since it's an internal route. `fullProfilePage` breaks the link on local dev since it uses env-based protocol/host.